packages/console/src/console.ts

```diff
@@ -0,0 +1,95 @@
+import * as REPL from 'repl'
+import path from 'path'
+import fs from 'fs'
+import {REPLCommand, REPLServer} from 'repl'
+import {watch} from 'chokidar'
+import pkgDir from 'pkg-dir'
+import os from 'os'
+
+// import {loadDependencies} from '../utils/load-dependencies'
+import {getBlitzModulePaths, loadBlitz} from './utils/load-blitz'
+
+const projectRoot = pkgDir.sync() || process.cwd()
+
+const commands = {
+  reload: {
+    help: 'Reload all modules',
+    action(this: REPLServer) {
+      this.clearBufferedCommand()
+      console.log('Reloading all modules...')
+      loadModules(this)
+      this.displayPrompt()
+    },
+  },
+}
+
+const runConsole = async (replOptions: REPL.ReplOptions) => {
+  const repl = initializeRepl(replOptions)
+
+  const watchers = [
+    // watch('package.json').on('change', () => Console.loadDependencies(repl)),
+    watch(getBlitzModulePaths()).on('all', () => loadBlitzModules(repl)),
+  ]
+
+  repl.on('reset', () => loadModules(repl)).on('exit', () => watchers.forEach((watcher) => watcher.close()))
```

Comment:
According to the [repl](https://nodejs.org/api/repl.html#repl_commands_and_special_keys) reference there is no `.reset` command. There is only `.clear` and maybe it is a different command rather than reset. Besides, we have added `.reload` in the custom commands to do the same. So maybe we could remove the `.on('reset')` here.

Reply:
yeah we probably don't need `reset` here